api/routes/utils.js

```diff
@@ -511,8 +511,8 @@ var utils = {
 
     // Update all fields for the toUser
     var toUserReps = this.updateTransactionToUser(toUser, fromUser, category.name, transaction.amount);
-    if (!toUserReps) {
-      return 'User is not an expert for category';
+    if (toUserReps === null) {
```

Comment:
toUserReps hitting 0 from revoking all of the last investment was causing the error case to happen